docs/chap-networking.md

```diff
@@ -214,6 +214,158 @@ The Polkadot Host must actively communicate with the network in order to partici
+:::definition
+
+The `SignedAuthorityRecord` is a Protobuf serialized structure representing the authority records and signature to send over the wire.
+It is defined in the following format:
+
+| Type               | Id  | Description                                           |
+|--------------------|-----|-------------------------------------------------------|
+| *AuthorityRecord*  | 1   | Serialized authority record                           |
+| `bytes`            | 2   | An Schnorrkel/Ristretto x25519 ("sr25519") signature  |
+| *PeerSignature*    | 3   | Serialized peer signature                             |
+
+**where**  
+
+**AuthorityRecord** is a serialized Protobuf structure that lists the addresses of authority nodes that are currently part of the authority set. 
+
+| Type             | Id | Description                                                           |
+|------------------|----|-----------------------------------------------------------------------|
+| `repeated bytes` | 1  | Possibly multiple `MultiAddr`ess through which a node can be connected | 
+
+**PeerSignature** is a Protobuf serialized structure indicating the signature and public key used to sign and verify the `AuthorityRecord`.
+This is the protobuf structure used to exchange the signature with other nodes.
+
+| Type    | Id  | Description                                       |
+|---------|-----|---------------------------------------------------|
+| `bytes` | 1   | An sr25519 signature                              | 
+| `bytes` | 2   | A sr25519 public key used to verify the signature | 
+
+:::
+
+
+###### Definition -def-num- Publish authority addresses message {#defn-msg-auth-discovery-publish}
+:::definition
+
+For each authority node $i$ in the current authority set, the local node sends a `put_value` message to the Kademlia DHT in the following format:
+
+$$
+\texttt{put\_value}{\left(\texttt{KademliaKey}_{i} , \texttt{Sig}_{AR}\right)}
+$$
+
+
+
+**where**
+- $\texttt{KademliaKey}_{i}$ is the $Sha256$ hash of the authorityId of node $i$. 
+- $\texttt{Sig}_{AR}$ is the `SignedAuthorityRecord`
+ described above([Definition -def-num-ref-](chap-networking#defn-msg-signed-authority-record)). 
+:::
+
+
+
+#### -sec-num- Discovering authority addresses {#sect-auth-discovery-worker-discover}
+<!-- worker component -->
+The authority discovery mechanism periodically sends `get_value` messages to the Kedemlia DHT in order to discover addresses of authority nodes it knows about.
+
+###### Definition -def-num- Discover authority addresses message {#defn-msg-auth-discovery-lookup}
+:::definition
+
+At each cycle, nodes `worker` creates a bounded number of `get_value` messages to send in the following format:
+$$
+\texttt{get\_value}{\left(\texttt{KademliaKey}_{i}\right)}
+$$
+
+**where**
+- $\texttt{KademliaKey}_{i}$ is the $Sha256$ hash of the `authorityId` of node $i$ selected from the current authority set.
+::: 
+
```

Comment:
Again that's an implementation detail. We don't need to explain how to retrieve the list of authorities, that's an exercise for the reader.

Reply:
I agree that it is indeed straightforward, but providing added convenience for developers could be beneficial.